testproject/Assets/Tests/Runtime/MultiprocessRuntime/Helpers/MultiprocessOrchestration.cs

```diff
@@ -43,10 +66,13 @@ public static void StartWorkerNode()
             throw;
         }
 
+        string logPath = Path.Combine(s_MultiprocessDirInfo.FullName, $"zlogfile{Processes.Count}");
+
+
         workerProcess.StartInfo.UseShellExecute = false;
         workerProcess.StartInfo.RedirectStandardError = true;
         workerProcess.StartInfo.RedirectStandardOutput = true;
-        workerProcess.StartInfo.Arguments = $"{IsWorkerArg} -popupwindow -screen-width 100 -screen-height 100";
+        workerProcess.StartInfo.Arguments = $"{IsWorkerArg} -popupwindow -screen-width 100 -screen-height 100 -logFile {logPath}"; // Let each process log to a different file
```

Comment:
should we have popup windows or run these processes in -batchmode instead?

(perhaps not a great topic for this discussion but wanted to put it here anyway)

Reply:
I'm not sure, this is how it was constructed before. I didn't want to change what I didn't understand.

Reply:
fair enough but I'd like to summon @SamuelBellomo here as he's the original author :)

Reply:
good point, these arguments were added with local testing in mind, they should be adapted to headless when running in CI